lib/c-api/tests/wasm_c_api/wasm-c-api/README.md

```diff
@@ -94,6 +94,7 @@ Currently, known implementations of this API are included in
 * V8 natively (both C and C++)
 * Wabt (only C?)
 * Wasmtime (only C?)
+* [Wasmer](https://github.com/wasmerio/wasmer/tree/master/lib/c-api) (only C, C++ coming soon)
```

Comment:
Unrelated to this PR but I want to put this idea out into the universe before I forget:

When doing the release we have `c-api/docs/deprecated` and we currently package that as `README.md` in the C API that we distribute...

We should probably make a note that it's the deprecated API and file an issue to start building docs for Wasmer with the Wasm C API!

Reply:
Completely!